src/lib.rs

```diff
@@ -13,26 +13,17 @@
 
 #![crate_type = "lib"]
 #![cfg_attr(test, feature(test))]
-#![feature(proc_macro)]
-#![feature(fnbox)]
-#![feature(alloc)]
-#![feature(slice_patterns)]
-#![feature(box_syntax)]
-#![feature(integer_atomics)]
-#![feature(entry_or_default)]
-#![cfg_attr(feature = "dev", feature(plugin))]
-#![cfg_attr(feature = "dev", plugin(clippy))]
-#![cfg_attr(not(feature = "dev"), allow(unknown_lints))]
+#![feature(
```

Comment:
Is this re-arranged by fmt or manually? In fact I prefer previous styles so that I can easily add / remove one feature without touching the rest.